backend/uploader.py

```diff
@@ -76,10 +75,17 @@ def save_manifest_to_db(manifest):
             return manifest['_id']
         else:
             print(is_valid, file=sys.stderr)
-            validation_error = []
-            for error in sorted(validator.iter_errors(manifest), key=str):
+            errors = sorted(validator.iter_errors(manifest), key=str)
+            validation_error = {}
+            validation_error["upper_errors"] = []
```

Comment:
I would like this to be named just "errors" and "sub_errors", what do you think?